sovtokenfees/sovtokenfees/test/helpers/helper_request.py

```diff
@@ -24,131 +25,25 @@ class HelperRequest(token_helper_request.HelperRequest):
 
     def set_fees(self, fees):
         """ Build a request to set the fees. """
-        payload = {
-            TXN_TYPE: SET_FEES,
-            FEES: fees,
-        }
 
-        request = self._create_request(payload,
-                                       identifier=self._wallet._trustee_wallets[0].defaultId)
+        request = build_set_txn_fees_req(self._client_wallet_handle, self._wallet._trustees[0], 'sov', json.dumps(fees))
+        request = self._looper.loop.run_until_complete(request)
         request = self._wallet.sign_request_trustees(request, number_signers=3)
+        request = json.loads(request)
+        sigs = request["signatures"]
+        request = self._sdk.sdk_json_to_request_object(request)
+        setattr(request, "signatures", sigs)
```

Comment:
Why do we need these lines? Just to create Request object instead of json, since existing API expects Request objects?

Reply:
Current Request API erases signatures from json. We need to save it and set it in request.